### PR TITLE
add build frontend

### PR DESCRIPTION
**Pull request by JeroenKnoops**
Add test for frontend in build pipeline as well.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -10,10 +10,30 @@ env:
...
+    name: Build frontend project
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout 
+        uses: actions/checkout@v2
 
+      - uses: actions/setup-java@v2
+        with:
+          java-version: '15'
```
Comment (timovandeput) [inline .github/workflows/build.yml:23]: Is Java really required to build and test the Flutter front end?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -37,13 +57,10 @@ jobs:
 
       - name: Test with Maven
         run: mvn test
-      
-      - name: Install with Maven
-        run: mvn install
```
Comment (timovandeput) [inline .github/workflows/build.yml:42]: The "Build with Maven" step on line 55 not only installs, but I expect it also runs the unit tests.

(Suggestion is to add a `-DskipTests` to suppress the tests during the build.)

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -66,5 +83,5 @@ jobs:
         uses: philips-software/docker-ci-scripts@v3.2.1
         with:
           dockerfile: docker/Dockerfile
-          image-name: bom-bar 
+          image-name: bom-base
```
Comment (timovandeput) [inline .github/workflows/build.yml:86]: I think the image should still be called "bom-bar". :-)

**Review by timovandeput — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (JeroenKnoops) [reply .github/workflows/build.yml]: Removed it.. :)

File: .github/workflows/build.yml (hunk 2 above)
Comment (JeroenKnoops) [reply .github/workflows/build.yml:42]: I always make this mistake. I want a build and running of tests to be separate steps. I want to be able to see the difference between not compilable code and failing tests..
I've added `-DskipTests` in this workflow.

**Review by timovandeput — APPROVED** [review]
LGTM

**Pull request merged**